### PR TITLE
[Serializer] Mention adder method to deserialize an array of objects

### PR DESCRIPTION
**Pull request by alexandre-daubois**
After spending a whole day with a colleague to try to deserialize/denormalize an array of objects a few weeks ago without succeeding, I thought that I might share this "adder method" tip to everyone now that I finally found the solution... 😅

### CODE REVIEW

File: components/serializer.rst [hunk 1]
```diff
@@ -1751,6 +1751,61 @@ will be thrown. The type enforcement of the properties can be disabled by settin
 the serializer context option ``ObjectNormalizer::DISABLE_TYPE_ENFORCEMENT``
 to ``true``.
 
+It is also possible to denormalize an array of object. To do so, ensure
```
Comment (OskarStark) [inline components/serializer.rst:1754]:
```suggestion
It is also possible to denormalize an array of objects. To do so, ensure
```
?

File: components/serializer.rst [hunk 2]
```diff
@@ -1751,6 +1751,61 @@ will be thrown. The type enforcement of the properties can be disabled by settin
...
+the owner object contains a adder method, so the Serializer is able to
```
Comment (OskarStark) [inline components/serializer.rst:1755]:
```suggestion
the owner object contains an adder method, so the Serializer is able to
```

File: components/serializer.rst [hunk 3]
```diff
@@ -1751,6 +1751,61 @@ will be thrown. The type enforcement of the properties can be disabled by settin
...
+guess the type of object to denormalize in your collection. Let's
```
Comment (OskarStark) [inline components/serializer.rst:1756]:
```suggestion
guess the type of the object to denormalize in your collection. Let's
```

File: components/serializer.rst [hunk 4]
```diff
@@ -1751,6 +1751,61 @@ will be thrown. The type enforcement of the properties can be disabled by settin
...
+rename ``$inner`` to ``$inners`` and create a adder method::
```
Comment (OskarStark) [inline components/serializer.rst:1757]:
```suggestion
rename ``$inner`` to ``$inners`` and create an adder method::
```

File: components/serializer.rst [hunk 5]
```diff
@@ -1751,6 +1751,61 @@ will be thrown. The type enforcement of the properties can be disabled by settin
...
+            return $this->inners;
+        }
+
+        public function setInners(array $inners)
+        {
+            $this->inners = $inner;
+        }
+
+        // This is the key method that will allow the Serializer
+        // to know the type of object of your collection
```
Comment (OskarStark) [inline components/serializer.rst:1784]:
```suggestion
        // to know the type of the object of your collection
```

**Comment by wouterj** [comment]
As indicated in #17814, this article is going to be removed. Let's not add more text to it

**Comment by alexandre-daubois** [comment]
Oh yes sorry, I forgot about it. You're right. I think it worth mention it, feel free to take this PR's content if needed 😄

**Pull request closed without merging**